### ISSUE TITLE
Campfire Mode Does Not Turn Off

### ISSUE THREAD

**Issue by ShaunSHamilton** [freeCodeCamp/freeCodeCamp#46098]
### Describe the Issue

Initial report: https://forum.freecodecamp.org/t/cant-turn-off-campfire-mode/510250

It appears we are no longer setting the `fcc-sound` localStorage variable, when toggling the setting. However, it appears we still rely on the value from localStorage for the setting.

### Affected Page

`/learn`

### Steps to Reproduce

1. Go to https://freecodecamp.org/settings
2. Toggle Campfire Mode on
3. Ensure that it is on (might not be, if you have previously toggled it)
4. Toggle Campfire Mode off
5. Perform actions to see it is still on

### Expected behavior

It is my assumption we no longer want to be relying on the localStorage for this setting?

### System

N/A

**Comment by ojeytonwilliams** [comment]
> It is my assumption we no longer want to be relying on the localStorage for this setting?

Yep, we should just gett the value from the user object and use that wherever we currently use 'fcc-sound'.

**Comment by scissorsneedfoodtoo** [comment]
We got a similar report in the support inbox recently.

I tried with my account and wasn't able to toggle Campfire Mode on or off. My account started with Campfire Mode off, and when I turned it on in the settings and checked localStorage, `fcc-sound` was still set to `false`.

After manually setting the `fcc-sound` flag to true and turning Campfire Mode off in the settings, `fcc-sound` was still set to `true`.
